docs/guide/learning/how-to-learn-vue.md

```diff
@@ -243,6 +243,68 @@ In this section, we will go over the most popular technologies and the struggles
 
 ### Laravel
 
+<!-- TODO: Missing Link -->
+You may be interested in [checking out the dev.to article](). It mirrors this section entirely (heading for heading), going into greater detail, while providing code samples and demos.
+
+Otherwise, read on for a brief overview of why Laravel and Vue is a fantastic combination.
```

Comment:
Now you should give your one or two sentences on WHY this is a good idea, why using Vue is so nice with Laravel. 

Something like the one below, where you prepare the user in a way.

Laravel is a very popular choice for both API only and server rendered websites with Vue. The two are framework agnostic, making them a very good match. The wide community adoption also means more modules, plugins and answers for common problems.

Reply:
> Now you should give your one or two sentences on WHY this is a good idea, why using Vue is so nice with Laravel.
> 
> Something like the one below, where you prepare the user in a way.
> 
> Laravel is a very popular choice for both API only and server rendered websites with Vue. The two are framework agnostic, making them a very good match. The wide community adoption also means more modules, plugins and answers for common problems.

@sustained you can get the idea from my PR. Feel free to modify.
+1 for the article. Really liked it
```
**Combination**

- Laravel comes with builtin Vue example component
- Easy to manage Cookies, Authentication, Routing, Sessions and Caching
- Single codebase for frontend and backend

**Drawbacks**

- Worse SEO
- Define a catch-all 404 route on the frontend 

<useful-links>
<useful-links-section title="Useful links"> 

 [Building a Vue SPA with Laravel](https://laravel-news.com/using-vue-router-laravel)
 [Why should you use Vue.js when using Laravel](https://blog.pusher.com/why-vuejs-laravel/)

</useful-links-section>
</useful-links>
```